examples/logging_configuration.md

```diff
@@ -0,0 +1,234 @@
+# Logging Configuration
+
+vLLM leverages Python's `logging.config.dictConfig` functionality to enable
+robust and flexible configuration of the various loggers used by vLLM.
+
+vLLM offers two environment variables that can be used, individually or in
+combination, to accommodate a range of logging configurations that vary in
+flexibility from:
+
+- vLLM's default logging configuration (least flexible)
+- coarse grained custom logging configuration
+- fine-grained custom logging configuration (most flexible)
+
+
+## Logging Configuration Environment Variables
+
+### `VLLM_CONFIGURE_LOGGING`
+
+`VLLM_CONFIGURE_LOGGING` controls whether or not vLLM propagates configuration
+from the root vLLM logger ("vllm") to other loggers used by vLLM. This
```

Comment:
nit
```suggestion
from the root vLLM logger (logger.getLogger("vllm")) to other loggers used by vLLM. This
```

Reply:
This doc was removed when the docs were simplified due to `propagate=True` goodness.